x-pack/plugins/endpoint/public/embeddables/resolver/types.ts

```diff
@@ -174,9 +189,26 @@ export interface ProcessEvent {
     source_id?: number;
     process_name: string;
     process_path: string;
+    signature_status?: string;
   };
 }
 
+/**
+ * A map of Process Ids that indicate which processes are adjacent to a given process along
+ * directions in two axes: up/down and previous/next.
+ */
+export interface AdjacentProcessMap {
```

Comment:
Do you think it'd help making this interface a class and then putting some logic around how `level` is set? That way you could enforce it needing to be >= 1

Reply:
It's a simple enough structure that I didn't feel the need to extend it beyond this expression of shape, but we could do that.